src/Power.cpp

```diff
@@ -389,10 +389,8 @@ bool Power::analogInit()
 #else
     analogReference(AR_INTERNAL); // 3.6V
 #endif
-    analogReadResolution(BATTERY_SENSE_RESOLUTION_BITS); // Default of 12 is not very linear. Recommended to use 10 or 11
-                                                         // depending on needed resolution.
-
 #endif // ARCH_NRF52
+    analogReadResolution(BATTERY_SENSE_RESOLUTION_BITS);
```

Comment:
This breaks ESP32 architecture adc readings as it will set the ADC to 10bit when the default value for esp32 is set to 12bit. Fixed in [11d78be](https://github.com/meshtastic/firmware/pull/3163/commits/11d78be541809a2ebba8ea9d58b03619c0336e68)